tests/test_auditlog.py

```diff
@@ -1,5 +1,9 @@
+from django.contrib.contenttypes import management
```

Comment:
To run a management command you should be importing `from django.core.management import call_command` but if you just need test publications, we should extract test publication creation logic into the base test class.